packages/graphql/src/schema/validation/custom-rules/directives/default.ts

```diff
@@ -48,21 +50,38 @@ export function verifyDefault(enums: EnumTypeDefinitionNode[]) {
         }
 
         if (!isArrayType(traversedDef)) {
-            if (isSpatial(expectedType)) {
-                throw new DocumentValidationError(`@default is not supported by Spatial types.`, ["value"]);
-            } else if (isTemporal(expectedType)) {
+            if (["DateTime", "LocalDateTime", "Date"].includes(expectedType)) {
```

Comment:
Nitpick: IMO `[GraphQLDateTime.name, GraphQLLocalDateTime.name, GraphQLDate.name]`  is better to avoid potential typos.